src/lib.rs

```diff
@@ -286,6 +287,19 @@ impl Datadog<dogstatsd::Client> {
             instance.do_event(metric.as_ref(), text.as_ref(), tags);
         }
     }
+
+    /// Start measuring a particular path through a region of code being compared
+    /// When this guard is dropped, it will emit a timing metric for the duration it
+    /// existed. The timing data is emitted under the "experiments" metric, tagged with the
+    /// given tags, plus "name:<experiment_name>" and "path_taken:<path>". If more than
+    /// i64::MAX milliseconds have elapsed, the metric will also be tagged with "overflowed".
```

Comment:
Can we leave a note on how much time i64::MAX milliseconds is?

Reply:
Sure, will add